src/MBrace.Azure/Store/BlobStore.fs

```diff
@@ -213,37 +210,42 @@ type BlobStore private (account : AzureStorageAccount, defaultContainer : string
         member this.Name = "MBrace.Azure.Store.BlobStore"
         member this.Id : string = account.CloudStorageAccount.BlobStorageUri.PrimaryUri.ToString()
 
-        member this.DefaultDirectory = defaultContainer
+        member this.DefaultDirectory = defaultContainer |> defaultArg <| ""
```

Comment:
Please don't use ``x |> y <| z``.  A good rule is never to have both ``|>`` and ``<|`` on the same line.  (A better rule is almost never to use ``<|`` in code you want to be readable by others)